package.json

```diff
@@ -8,13 +8,15 @@
     "@testing-library/user-event": "^7.1.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1"
+    "react-scripts": "3.4.1",
+    "typescript": "^3.9.6"
```

Comment:
Do we need TypeScript for anything?

Reply:
No we do not must have mistakenly added it to the package.json file